charts/plex-media-server/templates/statefulset.yaml

```
@@ -26,6 +26,9 @@ spec:
         {{- toYaml .Values.nodeSelector | nindent 8 }}
       affinity:
         {{- toYaml .Values.affinity | nindent 8 }}
+      {{- if .Values.priorityClassName }}
+      priorityClassName: "{{ .Values.priorityClassName }}"
```

nit: preference is to pipe to `quote`, eg:
```suggestion
      priorityClassName: {{ .Values.priorityClassName | quote }}
```